data/sea/54-zebra.h

```diff
@@ -152,12 +209,13 @@ static int64_t zebra_translate_column
 
         case ZEBRA_ARRAY:
             {
+                int64_t offset = 1;
                 istring_t *strings = anemone_mempool_alloc (mempool, count * sizeof (istring_t));
-                int offset = 1;
                 for (int i = 0; i != count; ++i) {
-                    int len = data.a.n[i];
-                    offset = zebra_translate_table (mempool, elem_start, len, (void**)(strings + i), &data.a.table);
-                    elem_start += len;
+                    int64_t index = i + elem_start;
+                    int64_t start = data.a.n[index]   - data.a.n[0];
+                    int64_t len   = data.a.n[index+1] - data.a.n[index];
+                    offset = zebra_translate_array (mempool, start, len, (void**)(strings + i), &data);
```

Comment:
can we add an assertion that

```
data.a.table.columns_count == 1 &&
data.a.table.columns[0].type == ZEBRA_BYTE
```

and if it's false just die really badly saying we don't support that input format